File: 0001-two-sum/0001-two-sum.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        int temp;
        vector<int> answer;
        for(int i=0;i<nums.size();i++){
            temp=target-nums[i];
            for(int j=i+1;j<nums.size();j++){
                if(nums[j]==temp){
                    answer.push_back(i);
                    answer.push_back(j);
                    return answer;
                }
            }
        }
        return answer;
    }
};

File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        hash_map={}
        for num in range(len(nums)):
            hash_map[nums[num]]=num
        for num in range(len(nums)):
            diff=target-nums[num]
            if diff in hash_map and hash_map[diff] != num:
                return [num, hash_map[diff]]

        